src/parse_structure.ml

```diff
@@ -52,7 +52,12 @@ let type_with_init lst =
   try
     ret @@
       extract_typedtree @@
-        Typemod.type_structure (init_env ()) lst Location.none
+        Typemod.type_structure (init_env ()) lst
+#if OCAML_VERSION >= (4, 12, 0)
```

Comment:
Is it possible to use `#if OCAML_VERSION < (4, 12, 0)` to avoid the empty branch ?

Reply:
Sure !